cmd/gotk/create.go

```diff
@@ -31,10 +33,25 @@ var createCmd = &cobra.Command{
 var (
 	interval time.Duration
 	export   bool
+	labels   []string
 )
 
 func init() {
 	createCmd.PersistentFlags().DurationVarP(&interval, "interval", "", time.Minute, "source sync interval")
 	createCmd.PersistentFlags().BoolVar(&export, "export", false, "export in YAML format to stdout")
+	createCmd.PersistentFlags().StringSliceVar(&labels, "label", nil, "label in the format key=value, accepts comma-separated items")
```

Comment:
Helm uses something like the following to make the "comma-separated items" more clear to new (CLI) users:

```suggestion
	createCmd.PersistentFlags().StringSliceVar(&labels, "label", nil, "set labels on the resource (can specify multiple labels with commas: label1=value1,label2=value2)")
```

Reply:
Cool, updated!